### ISSUE TITLE
[2.1.20] iPhone 6 Plus / 10.2.1 - User is able to login with another username with logging out the existing username.

### ISSUE THREAD

**Issue by Pankaj-Testershub** [Lomotif/lomotif-ios-issues#1569]
Every time:

Precondition:
- Facebook account should be login in "Safari/Facebook" app

Action:
1: Launch the app.
2: Navigate to "Profile>>Sign up/login" screen and tap on "Connect with Facebook" option.
3: While loading process taps on "X" option at top right corner of the screen and navigate to "Profile" screen.
4: Immediately tap on "Sign up/Login" button and navigate to "Sign up/Login" screen.
5: Navigate to "Log in" screen and enter a valid credential.
6: Tap on "Login" button.
7: Check if the user is able to login with another username with logging-out the existing username.

Expected result: User should not be able to login with another username without logging out the existing username.

Actual result: User is able to login with another username without logging out the existing username.

Refer following video link for more details:
https://drive.google.com/open?id=0B0-oVt0Gfy3iMmdKeThLcThFRUk

**Comment by Aboli-Testershub** [comment]
@xiuie I can reproduce this issue for build [2.1.20] on iPhone 7 / 10.2.1

**Comment by xiuie** [comment]
@Pankaj-Testershub Isn't this expected behavior? If you click "X" you're cancelling facebook login.

**Comment by Pankaj-Testershub** [comment]
@xiuie No, if we tap on "X" and navigate to "Profile" screen and wait for few secs, the "Facebook" logging-in process gets completed and it is displayed as logged in to app with "Facebook" account.
